tensor2tensor/utils/trainer_lib.py

```diff
@@ -391,7 +391,8 @@ def run_std_server(self):
     server = tf.train.Server(
         config.cluster_spec,
         job_name=config.task_type,
-        task_index=config.task_id)
+        task_index=config.task_id,
+        protocol=self._hparams.protocol)
```

Comment:
This change seems to breaks with commit [730201a7d28e2ded76145ae7dd7a663311c86d4a](730201a7d28e2ded76145ae7dd7a663311c86d4a).  `self._hparams` is `None`.